ssg/entities/common.py

```diff
@@ -309,3 +349,85 @@ def update_with(self, rhs):
         updated_refinements = self._subtract_refinements(extended_refinements)
         updated_refinements.update(self.refine_rules)
         self.refine_rules = updated_refinements
+
+
+class Templatable(object):
```

Comment:
It would be great to add some doc text here to explain what "Templatable" is.

Reply:
Ont it.